### PR TITLE
Tim

### PR DESCRIPTION
**Pull request by thallaron**
I removed my unwanted datamanager.js and json file. 

Added early draft of todo.js management functions. Should the retrieve function return an array of javascript objects?

### CODE REVIEW

File: routes/todo.js [hunk 1]
```diff
@@ -15,10 +16,35 @@ module.exports = function (io) {
...
+            
 			console.log(data)
 		})
     })
     return router
 }
 
+//appends the object into tasks.txt
+function storeObject(task){
+    var stringLine = [task.name, task.start, task.stop, task.details, task.priority +'\n']
```
Comment (typecampo) [inline routes/todo.js]: you need to join them with stringLine.join(',') 
Not sure if we need the \n you will need to test it.

File: routes/todo.js [hunk 2]
```diff
@@ -15,10 +16,35 @@ module.exports = function (io) {
...
+    var jsObject
+    fs.readFile('message.txt', (err, data) => {
+        if (err) throw err;
+        console.log(data);
+      });
+
+
+
+    return jsObject
+}
```
Comment (typecampo) [inline routes/todo.js]: new line

File: routes/todo.js [hunk 3]
```diff
@@ -15,10 +16,35 @@ module.exports = function (io) {
...
+            fs.appendFile('tasks.txt', stringLine, (err) => {
+                if (err) throw err;
+                console.log('The stringLine was appended to file!');
+              });
+}
+
+//ideally retreives objects from tasks.txt. Should this return an array of tasks?
+function retrieveObject(p1){
+    var jsObject
+    fs.readFile('message.txt', (err, data) => {
```
Comment (typecampo) [inline routes/todo.js]: need it to return an array of each line

**Comment by thallaron** [comment]
My storeObject function doesn't seem to be getting called. Which is weird because it worked last week.

**Comment by typecampo** [comment]
If you pulled from master, the socket.emit('task') function is commented out. I am going to send you an array of objects. Then on the server, loop through the array and store each object.

**Comment by typecampo** [comment]
Implemented readFile and appendFile

**Pull request closed without merging**